tests/test_augmentations.py

```diff
@@ -179,7 +184,8 @@ def test_augmentations_wont_change_input(augmentation_cls, params, image, mask):
     image_copy = image.copy()
     mask_copy = mask.copy()
     aug = augmentation_cls(p=1, **params)
-    aug(image=image, mask=mask)
+    annotations = {'image': image, 'cropped_bbox': [-59, 77, 177, 231]}
```

Comment:
you should not change tests, just add another if existing does not fit your needs.